pkg/controller/cloud/node_lifecycle_controller.go

```diff
@@ -106,6 +106,7 @@ func NewCloudNodeLifecycleController(
 // be called via a goroutine
 func (c *CloudNodeLifecycleController) Run(stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
+	klog.Info("Starting CloudNodeLifecycleController")
```

Comment:
ditto - this line seems for debugging purpose only.

Reply:
This one happened but not CloudNodeController, not sure their relationship. Prefer to keep the log as this is not repeatable on local dev machine.